proto/v3/message_contents/message.proto

```diff
@@ -4,33 +4,42 @@ syntax = "proto3";
 package xmtp.v3.message_contents;
 
 import "v3/message_contents/association.proto";
+import "v3/message_contents/encoded_content.proto";
 
 option go_package = "github.com/xmtp/proto/v3/go/v3/message_contents";
 option java_package = "org.xmtp.proto.v3.message.contents";
 
-// Metadata that is encrypted via SealedSender and only visible to the recipient
-// Currently we do not actually encrypt this, actual implementation of
-// SealedSender will be added shortly.
-message PadlockMessageSealedMetadata {
-  string sender_user_address = 1;
-  string sender_installation_id = 2;
-  string recipient_user_address = 3;
-  string recipient_installation_id = 4;
-  bool is_prekey_message = 5;
-}
-
 // Plaintext header included with messages, visible to all
 // Recipients can verify this header has not been tampered with.
 // Servers are unable to verify if the header has been tampered with.
-message PadlockMessageHeader {
+message MessageHeaderV1 {
+  // SealedSender wrapper for arbitrary bytes
+  message SealedSender {
+    bytes ephemeral_public_key = 1;
+    bytes ephemeral_static_key = 2;
+    bytes encrypted_message = 3; // Will be serialized Metadata
+  }
+
+  // Metadata that is encrypted via SealedSender and only visible to
+  // the recipient
+  message Metadata {
+    string sender_user_address = 1;
+    string sender_installation_id = 2;
+    string recipient_user_address = 3;
+    string recipient_installation_id = 4;
+    bool is_prekey_message = 5;
+  }
```

Comment:
Are we feeding this whole blob into AES as the equivalent of the `senderIdentityPublic` input in https://signal.org/blog/sealed-sender/, or as the equivalent of the `sender_certificate`?